From d5092ae26f72fd9c40b65484d3684dbf111733a7 Mon Sep 17 00:00:00 2001
From: Nicolas LOUBOUTIN <nicolas.louboutin@st.com>
Date: Wed, 16 Sep 2020 11:05:16 +0200
Subject: [PATCH] Remove CFI_CLANG and LTO_CLANG required configuration for GK
 5.4

CONFIG_CFI_CLANG and CONFIG_LTO_CLANG are not available for ARM architecture

Signed-off-by: Nicolas LOUBOUTIN <nicolas.louboutin@st.com>
Change-Id: I8ed5ef29e0261aa93a9fec523d65b538556fcc9c
---
 r/android-5.4/android-base.config | 2 --
 1 file changed, 2 deletions(-)

diff --git a/r/android-5.4/android-base.config b/r/android-5.4/android-base.config
index 08cfbb9..c78e8e1 100644
--- a/r/android-5.4/android-base.config
+++ b/r/android-5.4/android-base.config
@@ -29,7 +29,6 @@ CONFIG_BLK_DEV_LOOP=y
 CONFIG_BLOCK=y
 CONFIG_BPF_JIT=y
 CONFIG_BPF_SYSCALL=y
-CONFIG_CFI_CLANG=y
 CONFIG_CGROUPS=y
 CONFIG_CGROUP_BPF=y
 CONFIG_CGROUP_CPUACCT=y
@@ -109,7 +108,6 @@ CONFIG_IP_NF_TARGET_REDIRECT=y
 CONFIG_IP_NF_TARGET_REJECT=y
 CONFIG_JOYSTICK_XPAD=y
 CONFIG_L2TP=y
-CONFIG_LTO_CLANG=y
 CONFIG_MAGIC_SYSRQ=y
 CONFIG_MD=y
 CONFIG_MEMBARRIER=y
-- 
2.17.1

